packages/core/src/components/Label/__stories__/label.stories.tsx

```diff
@@ -56,6 +56,27 @@ export const Kinds = {
   }
 };
 
+export const Sizes = {
+  render: () => (
+    <>
+      <div className="monday-storybook-label_group monday-storybook-label_states-gap">
+        <Label text="New" />
+      </div>
+      <div className="monday-storybook-label_group monday-storybook-label_states-gap">
+        <Label text="New" size="small" />
+      </div>
```

Comment:
To avoid the divs appear in the code snippet in storybook you can use a decorator, you can see some examples in `TextArea.stories.tsx`. It will make it less cluttered